services/viva/microservice/src/lambdas/syncWorkflow.ts

```diff
@@ -84,41 +107,41 @@ function updateCaseDetailsWorkflow(keys: CaseKeys, newWorkflow: VivaWorkflow) {
 export async function syncWorkflow(input: LambdaRequest, dependencies: Dependencies) {
   const { personalNumber } = input.detail.user;
 
-  const userCases = await dependencies.getSubmittedOrProcessingOrOngoingCases(personalNumber);
-
-  const caseList = userCases.Items;
-
-  const isEmptyCaseList = caseList?.length === 0;
-
+  const caseList = await dependencies.getCaseList(personalNumber, [
+    ACTIVE_ONGOING,
+    ACTIVE_SUBMITTED,
+    ACTIVE_PROCESSING,
+    CLOSED,
+  ]);
+  const isEmptyCaseList = caseList.length === 0;
```

Comment:
Originally `caseList` could potentially be `undefined` - is this still true?

Reply:
Hmm. Fixar detta det tro?
`return (result.Items ?? []) as CaseItem[];`

Reply:
@dannil76 Det kommer att lösa det.